core/src/test/java/org/elasticsearch/common/rounding/TimeZoneRoundingTests.java

```diff
@@ -529,19 +525,30 @@ private static DateTimeUnit randomTimeUnit() {
         return DateTimeUnit.resolve(id);
     }
 
-    private static String toUTCDateString(long time) {
-        return new DateTime(time, DateTimeZone.UTC).toString();
-    }
-
-    private static long utc(String time) {
-        return time(time, DateTimeZone.UTC);
-    }
-
     private static long time(String time) {
-        return ISODateTimeFormat.dateOptionalTimeParser().parseMillis(time);
+        return time(time, DateTimeZone.UTC);
     }
 
     private static long time(String time, DateTimeZone zone) {
         return ISODateTimeFormat.dateOptionalTimeParser().withZone(zone).parseMillis(time);
     }
+
+    private static Matcher<Long> isDate(final long expected, DateTimeZone tz) {
+        return new TypeSafeMatcher<Long>() {
+            @Override
+            public boolean matchesSafely(final Long item) {
+                return expected == item;
```

Comment:
I am unsure how boxing works in that case, it may be safer to unbox `item` explicitly?